### PR TITLE
Adds OSS Search Functionality to VS Code Extension

### PR DESCRIPTION
**Pull request by kevin-quigley**
1. Package.json
	a. Added an activation event
	b. Added a command and title
2. extension.ts
	a. Added async function and changed it to include Promise<void> since there was an error without it
	b. Registered the command under the async function. I have an error that it needs a void operator

### CODE REVIEW

File: extension.ts [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+			`https://sourcegraph.com/search?patternType=literal` +
+				`&q=${encodeURIComponent(value)}`
+		);
+}
+
+vscode.commands.registerCommand('extension.ossSearch', () => {
+	vscode.window.showInputBox().then(
+		ossSearchCommand
+	);
+});
```
Comment (tjkandala) [inline extension.ts:104]: Can we register this command the same way that we register our existing commands?
- register on activation in the `activate` function, add to `context.subscriptions` (read [this](https://stackoverflow.com/a/55579618) for more context on why this matters. we have [the same concept](https://docs.sourcegraph.com/extensions/authoring/activation#deactivation) for Sourcegraph extensions)
- wrap the command function with `handleCommandErrors`

**Review by tjkandala — COMMENTED** [review]
This is really cool @kevin-quigley! I'll definitely use this feature.

We should keep the code in [`src/extension.ts`](https://github.com/sourcegraph/sourcegraph-vscode/blob/master/src/extension.ts#L91-L92) since I believe that's what is compiled

**Comment by emidoots** [comment]
Ah thanks for reviewing TJ! I think he meant to send this as a draft PR as
we were planning to go over this and review it live in our next pair
programming session :) I’ll make sure we fix your comments!

-- 
Follow me on twitter @slimsag <https://twitter.com/slimsag>.

**Comment by kevin-quigley** [comment]
Whoops, apologies TJ! This is my first real PR, so I'm still learning the ropes. I'll coordinate with Stephen and find the right solution Wednesday during our 1:1

**Comment by emidoots** [comment]
Update: We followed up on all the feedback here and sent+merged a new PR #200

**Pull request closed without merging**